common/db.go

```diff
@@ -117,8 +135,26 @@ func InitDB(dbPath string) (*DB, error) {
 		// all good; nothing to do
 	}
 
+	// select all orchestrators updated in the last 24 hours
+	stmt, err := db.Prepare("SELECT serviceURI, ethereumAddr FROM orchestrators WHERE updatedAt >= datetime('now','-1 day')")
+	if err != nil {
+		glog.Error("Unable to select orchestrators updated in the past 24 hours", err)
+		d.Close()
+		return nil, err
+	}
+	d.selectOrchs = stmt
+
+	// updateOrchestrators statement
+	stmt, err = db.Prepare("INSERT OR REPLACE INTO orchestrators(updatedAt, serviceURI, ethereumAddr) VALUES(datetime(), ?, ?)")
```

Comment:
According to [this tutorial](http://www.sqlitetutorial.net/sqlite-replace-statement/) we can shorten the `INSERT OR REPLACE` statement to simply `REPLACE`. wdyt?

Reply:
Plus, I didn't know about the `REPLACE` feature until just now - thanks for this new learning!

Reply:
Sounds good to me!

Reply:
REPLACE alone does a delete-and-insert which might not necessarily always be what we want. For example, we store a creation timestamp here, which would be lost on a replace. It's not critical to keep that right now, but I can see us wanting to retain other types of fields in the future.

This brings up the fact that the query doesn't seem quite well formed, so thanks for bringing this up. To retain upsert-type behavior we probably need a nested select for the insertion time (and any other fields we wish to preserve). See https://stackoverflow.com/a/4330694

The latest SQLIte does seem to have native upsert as a brand-new feature, but I'm not sure what that means as far as distributing an up-to-date sqlite implementation (IIRC SQLite is typically a system lib but it could well be bundled somewhere in the go-sqlite implementation; can dig into this later). https://www.sqlite.org/draft/lang_UPSERT.html

Reply:
Update - doing some research on how we can make that work. Getting an error that I'm trying to fix locally... 

```db: Unable to fetch orchestrator sql: Scan error on column index 1, name "ethereumAddr": unsupported Scan, storing driver.Value type <nil> into type *string```

Reply:
>  I'm not sure what that means as far as distributing an up-to-date sqlite implementation 

Looks like we're in luck and go-sqlite **does** incorporate [its own copy of sqlite](https://github.com/mattn/go-sqlite3/commit/554bc0fb2e0ee7ec0105f4296b7fa5e01f85aa32), and the repo looks to be fairly up-to-date; we just need to refresh the goclient's vendored copy. This also partially explains why the goclient takes so long to build/link...

Doesn't to be done in this PR though; only if we really want to use the new upsert syntax here.

Reply:
🙀 thank you @j0sh! Adding `createdAt` and numbering the ?'s is genius. Why is it better to select on`createdAt` here instead of `ethereumAddr`? 
```sql
(SELECT createdAt FROM orchestrators WHERE ethereumAddr = ?2))")

Reply:
Makes sense. Thank you!